roles/satellite-clone/tasks/main.yml

```diff
@@ -1,10 +1,15 @@
 ---
-- include_vars: main.yml
-
 # include pre_install_check
 - include: pre_install_check.yml
   when: run_pre_install_check
 
+- name: "check that variables are updated"
+  fail: msg="Please update the variables in satellite-clone-vars.yml (after copying satellite-clone-vars.sample.yml)"
+  when:
+    - hostname == "satellite.example.com"
+    - activationkey == "changeme"
+    - org == "111111"
```

Comment:
I believe Ansible's when does a logical "and" here, which would imply that all of these conditions have to be true to fail. Thus, a user could change only one of these and get past this point. I think you want to swap to using "or" here?